conans/client/generators/virtualenv.py

```diff
@@ -1,9 +1,93 @@
 import os
+import platform
+import textwrap
+
+from jinja2 import Template
 
 from conans.client.tools.oss import OSInfo
 from conans.model import Generator
 
 
+sh_activate_tpl = Template(textwrap.dedent("""
+    #!/usr/bin/env bash
+    # DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+    # DIR="$( cd "$( dirname "$_" )" >/dev/null 2>&1 && pwd )"
+
+    {%- for it in modified_vars %}
+    export OLD_{{it}}="${{it}}"
```

Comment:
possible collisions? do we care? `CONAN_OLD_{{it}}`?

Reply:
I can add the `CONAN_` prefix 👍